frontend/src/v5/ui/routes/dashboard/projects/header/header.styles.ts

```diff
@@ -19,20 +19,12 @@ import styled from 'styled-components';
 
 export const Wrapper = styled.div`
 	width: 100%;
-	height: 114px;
-	padding: 0 30px;
+	height: 50px;
 	box-sizing: border-box;
 	background-color: ${({ theme }) => theme.palette.primary.contrast};
-	border: 1px solid ${({ theme }) => theme.palette.base.lightest};
-	border-left-style: none;
-	border-right-style: none;
-`;
-
-export const Container = styled.div`
-	display: flex;
-	flex-direction: column;
-	justify-content: space-between;
-	width: 100%;
-	max-width: 1289px;
-	margin: 28px auto 0;
+	/* 14px for home-icon left-padding + left-margin */
+	/* 15px breadcrumbs left-margin */
+	/* 30px 3D logo width */
+	/* 20px header left-padding */
```

Comment:
If we are not using these styles, just delete theme

Reply:
deleted